.golangci.yml

```diff
@@ -7,3 +7,9 @@ run:
 linters-settings:
   errcheck:
     exclude: .errcheck.txt
+  gofmt:
+    simplify: false
```

Comment:
what does this do? (simplifying sounds like a good thing?)

Reply:
Oh, I mistaken.

I realized that there is no problem in using simplify, but this line is deleted because `simplify: true` is an option of default.

gofmt: [The simplify command ¶](https://golang.org/cmd/gofmt/#hdr-The_simplify_command)
golangci: [#gofmt](https://github.com/golangci/golangci-lint#disabled-by-default-linters--e--enable)